boards/openmote-b/doc.txt

```diff
@@ -0,0 +1,59 @@
+/**
+@defgroup    boards_openmote-b OpenMote-b
```

Comment:
that is already defined in `boards/openmote-b/include/board.h` hence you need to remove it there an adapt the the `@ingroup` there accordingly too

Reply:
Yup, now matches the openmote-cc2538 way.